app/assets/javascripts/analytics/_events.js

```diff
@@ -97,19 +97,32 @@
         });
       }
     },
+    'opportunityDataDownload': function (e) {
```

Comment:
I don't really understand everything going on in the tracking events stuff, but it looks to me like the previous `supplierListDownload` event will be fired for *any* csv download?  So if we're introducing a new one here then maybe we need to adjust the previous event to be more choosy about when it thinks the supplier list has been downloaded?

Reply:
I could be wrong though.

Reply:
I will test it :)

Reply:
Ok, wrote tests, it turned out the old event tried to fire for opportunity data download and threw an error because the correct path was needed to create event label. To avoid this I have added and URL check before the event tries to fire, the same as new event has. This passes the tests and hopefully should solve the problem :).